sdk/storage/azblob/LICENSE

```diff
@@ -0,0 +1,21 @@
+    MIT License
```

Comment:
Nit: this should be LICENSE.txt file

Reply:
Done. We ported the same code we have in our current SDK. Hence the naming. Thanks for pointing out.